packages/next/server/next-dev-server.ts

```diff
@@ -196,6 +196,12 @@ export default class DevServer extends Server {
 
   async prepare() {
     await verifyTypeScriptSetup(this.dir, this.pagesDir!)
+    this.customRoutes = await this.getCustomRoutes()
+    const { redirects = [], rewrites = [] } = this.customRoutes
```

Comment:
Why do we set defaults to an empty array here? Wouldn't this already always be defined?
```suggestion
    const { redirects, rewrites } = this.customRoutes
```